.fossa.yml

```diff
@@ -280,6 +280,12 @@ targets:
     - type: gradle
       path: ./
       target: ':instrumentation:scala-fork-join-2.8:javaagent'
+    - type: gradle
+      path: ./
+      target: ':instrumentation:sofa-rpc-5.4:javaagent'
+    - type: gradle
+      path: ./
+      target: ':instrumentation:sofa-rpc-5.4:library-autoconfigure'
```

Comment:
the name `autoconfigure` here doesn't follow the normal pattern. Seems like it should just be `library` ?